### PR TITLE
Implementation of threshold

### PR DESCRIPTION
**Pull request by humleflue**
No compiler optimization as far as I can see
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # P6 - Deep Neural Network On Low Consumption Devices
```
Comment (christopher-jensen) [inline README.md:1]:
```suggestion
# P6 - Support Vector Classifier On Low Consumption Devices
```

File: c-svc23Vectors/math.h [hunk 2]
```diff
@@ -3,8 +3,9 @@
 
 #define EULER 2.71828
```
Comment (christopher-jensen) [inline c-svc23Vectors/math.h:4]: irrelevant now?

**Review by christopher-jensen — APPROVED** [review]
Looks good
I have two comments which is just questions primarily

File: c-svc23Vectors/math.h (hunk 2 above)
Comment (humleflue) [reply c-svc23Vectors/math.h:4]: Tjaaaaa. The whole radial kernel part is irrelevant now I guess, so we could delete it. I don't see anything wrong with keeping it there though, as it is a nice argument for future works/experiments 😉

**Pull request merged**